### DISCUSSION TITLE
如何调整zotero pdf阅读器 添加文字 的默认字体大小？

### DISCUSSION THREAD

**Discussion by reoLantern** [MuiseDestiny/zotero-style#1142] [Q&A]
![image](https://github.com/user-attachments/assets/ef3e709f-d4b3-484d-9cae-69566a8b3dbc)
指的是通过这里为pdf添加文字时，是否有办法设置添加文字的默认字体大小呢？
个人感觉这里默认添加文字太大，每次都需要手动缩放一下，比较麻烦，不知道是否有办法更改默认添加的字体大小。
谢谢大佬们

**Comment by XiongPan826** [comment]
同问

**Comment by reoLantern** [comment]
@MuiseDestiny

**Comment by MuiseDestiny** [comment]
无法调整，插件也没有做这一块。建议向官方提建议。

**Comment by jixingwu** [comment]
![image](https://github.com/user-attachments/assets/a4ba1253-9efc-48d0-ab1a-c2de36c93fef)
这里可以调整默认字体大小，但最小调整到10

**Reply by Vossera** [reply]
你这个回答不就是解决方案吗，赞一个，亲测有效

**Reply by yiwushan** [reply]
我测试无法默认，每次新增都要手动修改

**Comment by cjjcn** [comment]
https://forums.zotero.org/discussion/123091/request-for-automatic-line-break-configuration-in-pdf-reader-text-annotations#latest

我向官方提出了完善建议，但是目前还没有回复，大家可以一起呼吁下，引起官方关注。
I've made suggestions to the officials to make things better, but haven't heard back. Perhaps if we all voice our support, we can get the officials to take notice.

**Comment by zuoy3964** [comment]
同问
